manifests/y/Yandex/Chats/YandexMessenger/2.170.0/Yandex.Chats.YandexMessenger.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Yandex.Chats.YandexMessenger

```

```suggestion
PackageIdentifier: Yandex.Messenger
```

PackageId should be kept concise where possible.